app/src/components/tables/ValidatorsTable/ValidatorsTable.tsx

```diff
@@ -44,6 +46,23 @@ export const ValidatorsTable: React.FC = () => {
 
   const { t } = useTranslation();
 
+  const { type: themeType } = useTheme();
+
+  const blockExplorerTheme =
```

Comment:
Why do we need this? Theme as applied to css should be automatically toggled at the App level. In the styled component we should be able to do something like `color: ${props => props.theme.text.secondary};` for example, and everything should be handled under the hood.

Reply:
This is due to how our UI kit Table has been built.  We could think about adding a Provider to UI Kit to support this in the future.  Maybe let's chat next week

Reply:
Alrighty, sounds good 👍